cms/templates/cms/home_page.html

```diff
@@ -8,6 +8,26 @@
 
 {% block title %}{% trans "MIT MicroMasters" %}{% endblock %}
 
+{% block social %}
+  <meta property="og:site:name" content="MIT Micromasters">
+  <meta property="og:type" content="website">
+  <meta property="og:url" content="http://micromasters.mit.edu/">
```

Comment:
Can you change `http` to `https` here? Once we launch all access will be over HTTPS